### PR TITLE
Added editorial examples for text alternative elements in the termino…

### PR DESCRIPTION
**Pull request by richschwer**
…logy section to address issue 1019

### CODE REVIEW

File: accname-aam/accname-aam.html [hunk 1]
```diff
@@ -472,6 +472,7 @@
 			<h2>Substantive changes since the <a href="http://www.w3.org/TR/2014/WD-accname-aam-1.1-20141211/">last public working draft</a></h2>
 			<ul>
 				<li>08-June-2017: Remove list style information from the accessible name computation.</li>
+				<li>08-June-2017: Editorial change give example of text alternative element of &lt;label&gt; in the terminology sections 2A and 2F.</li>
```
Comment (joanmarie) [inline accname-aam/accname-aam.html]: If this is a purely editorial change, I think that means it is not a substantive change. (In which case, does this entry belong?)

File: accname-aam/accname-aam.html [hunk 2]
```diff
@@ -242,7 +242,7 @@
         <li id="step1">Initialize:  Set the <code>root node</code> to the given element, the <code>current node</code> to the <code>root node</code>, and the <code>total accumulated text</code> to the empty string ("").</li>
         <li id="step2">Compute the text alternative for the <code>current node</code>:
           <ol>
-            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor referenced by a native host language text alternative <a class="termref">element</a> or <a class="termref">attribute</a>, return the empty string.
+            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor referenced by a native host language text alternative <a class="termref">element</a> (e.g. <code>&lt;label&gt;</code> in HTML) or <a class="termref">attribute</a>, return the empty string.
```
Comment (joanmarie) [inline accname-aam/accname-aam.html]: Nit: Do we want the angle brackets? From a quick search, I only found one reference to a specific element (`strong`). But it's sans angle brackets. And the ARIA spec is sans angle brackets. But I see the Core AAM has an instance with angle brackets. So &lt;insert shrug here&gt;. Were it me, I'd remove the angle brackets. But if you feel that they should be there, that's fine.

File: accname-aam/accname-aam.html [hunk 3]
```diff
@@ -325,7 +325,7 @@
                 <pre class="example highlight"><code>&lt;div role="checkbox" aria-checked="false"&gt;Flash the screen &lt;span role="textbox" aria-multiline="false"&gt; 5 &lt;/span&gt; times&lt;/div&gt;</code></pre>
               </details></div>
               </li>
-            <li id="step2F">Otherwise, if the <code>current node's</code> <a class="termref">role</a> allows <a class="specref" href="#namefromcontent">name from content</a>, or if the <code>current node</code> is referenced by <code>aria-labelledby</code>, <code>aria-describedby</code>, or is a native host language text alternative <a class="termref">element</a>:
+            <li id="step2F">Otherwise, if the <code>current node's</code> <a class="termref">role</a> allows <a class="specref" href="#namefromcontent">name from content</a>, or if the <code>current node</code> is referenced by <code>aria-labelledby</code>, <code>aria-describedby</code>, or is a native host language text alternative <a class="termref">element</a> (e.g. <code>&lt;label&gt;</code> in HTML):
```
Comment (joanmarie) [inline accname-aam/accname-aam.html]: Ditto.

File: accname-aam/accname-aam.html (hunk 1 above)
Comment (richschwer) [inline accname-aam/accname-aam.html]: It was to address an issue Cynthia had and she asked for clarity by providing an example

**Pull request merged**